service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -110,13 +110,15 @@ public void persistHiddenDirectionValues(List<Element<Direction>> directionWithV
                     directionToAddValue.getValue().getAssignee(), direction.getValue().getAssignee()));
 
                 if (!direction.getValue().getReadOnly().equals("No")) {
-                    directionToAddValue.getValue().setDirectionText(direction.getValue().getDirectionText());
+                    directionToAddValue.getValue().setDirectionText(
+                        direction.getValue().getDirectionText().replace("’", "'"));
                 }
             }));
     }
 
     private boolean hasSameDirectionType(Element<Direction> directionToAddValue, Element<Direction> direction) {
-        return direction.getValue().getDirectionType().equals(directionToAddValue.getValue().getDirectionType());
+        return direction.getValue().getDirectionType()
```

Comment:
Does direction need the same treatment?

Reply:
If so it might be useful extracting the replacement to a private method

Reply:
no, so this is only actually a problem for the one case that Jason is drafting at the moment. The fix we pushed for christmas trees would not have corrected any data which has curly apostrophes / bullet points for any cases which are currently drafting an sdo.

Reply:
🎄